### ISSUE TITLE
Allow me to open two or more readings lists at the same time, so that I can compare and copy and paste between them

### ISSUE THREAD

**Issue by equivalentideas** [e-e-e/hyper-reader#36]
I wanted to compare to lists :) It intuitively came to me that they should open in new windows—worth noting, but I dunno if that's a good solution.

**Comment by sdockray** [comment]
I just moved house so only blindly jumping into these discussions (with potentially awful ideas) but if there is a side-by-side comparison going on, it would also be great to be able to drag and drop things from one list into another

Luke Bacon wrote:

**Comment by equivalentideas** [comment]
> if there is a side-by-side comparison going on, it would also be great to be able to drag and drop things from one list into another

Nice one @sdockray , I changed the description of this issue to reflect this:

> Allow me to open two or more readings lists at the same time, so that I can compare, and move content, between them

**Comment by e-e-e** [comment]
Yeah - actually at the moment you can open up multiple windows - its a bit funky as it opens completely new peer connections for each reading list - but you just need to `cmd+N`.

Diff'ing is also totally on the agenda.
